src/main/resources/data-repository-openapi.yaml

```diff
@@ -362,13 +471,13 @@ definitions:
       - columns
     properties:
       name:
-        $ref: '#/definitions/IdentifierNameModel'
+        $ref: '#/definitions/ObjectName'
       is_root:
         type: boolean
       columns:
         type: array
         items:
-          $ref: '#/definitions/IdentifierNameModel'
+          $ref: '#/definitions/ObjectName'
```

Comment:
This comment really applies to the asset description - we can't use the * character to indicate all columns because it does not conform to ObjectName.

Reply:
Good catch. Perhaps use the empty array to mean all of the columns? I don't think leaving out all columns is sensible.

Reply:
👍